### PR TITLE
make attach file working

### PR DESCRIPTION
**Pull request by darshanitaliya7110**
### What this does
Contact us - make attach file working and pass in payload

### Implementation
make attach file working

### Testing
(https://share.teamcamp.app/gDIMp7KjEjf5HH3LP6j0w)

### CODE REVIEW

File: components/contactUsModal/contactUsModal.tsx [hunk 1]
```diff
@@ -44,34 +51,55 @@ function ContactUsModal({ isSuccessfull, onClose }: Props) {
...
-    (event: React.ChangeEvent<HTMLInputElement>) => {
+    async (event: React.ChangeEvent<HTMLInputElement>) => {
       if (event.target.files) {
         const selectedFiles = Array.from(event.target.files);
+        const fileURL = await getFirebaseUrlFromFile(
+          selectedFiles,
+          'Contact-us',
+          selectedFiles[0]?.name,
+        );
         setFiles((prevFiles) => [...prevFiles, ...selectedFiles]);
```
Comment (aniket-pixer) [inline components/contactUsModal/contactUsModal.tsx:62]: [...(prevFiles || []), ...(selectedFiles || [])]

File: components/contactUsModal/contactUsModal.tsx [hunk 2]
```diff
@@ -44,34 +51,55 @@ function ContactUsModal({ isSuccessfull, onClose }: Props) {
...
+        if (fileURL) {
+          setAttachFiles((prev) => [
+            ...prev,
```
Comment (aniket-pixer) [inline components/contactUsModal/contactUsModal.tsx:65]: ...(prev || []),

File: components/contactUsModal/contactUsModal.tsx (hunk 1 above)
Comment (darshanitaliya7110) [reply components/contactUsModal/contactUsModal.tsx]: Done

File: components/contactUsModal/contactUsModal.tsx (hunk 2 above)
Comment (darshanitaliya7110) [reply components/contactUsModal/contactUsModal.tsx]: Done

**Pull request merged**